test/sonic_client_test.exs

```diff
@@ -16,27 +16,27 @@ defmodule ElixirSonicClientTest do
 
   test "ping" do
     {:ok, conn} =
-      ElixirSonicClient.start(
+      SonicClient.start(
         host(),
         1491,
         "search",
         "SecretPassword"
       )
 
-    assert {:ok, "PONG"} == ElixirSonicClient.ping(conn)
+    assert {:ok, "PONG"} == SonicClient.ping(conn)
     TcpConnection.close(conn)
   end
 
   test "stop connection" do
     {:ok, conn} =
-      ElixirSonicClient.start(
+      SonicClient.start(
         Kernel.to_charlist("sonic"),
```

Comment:
Did we forget to replace this with a call to `host()`, here?

Reply:
This was from your branch, and this branch wasn't up to date with main

Reply:
✌️

Reply:
I suspected this, this is why I used the "we" word. 😬